_data/ispn.yml

```diff
@@ -12,18 +12,18 @@
 # -----------------------------
 stable:
   minor_version: 12.1
-  version: 12.1.1.Final
-  maven_latest: 12.1.1.Final
-  release_date: 2021/04/14
-  fixed_issues: https://issues.redhat.com/secure/ReleaseNote.jspa?projectId=12310799&version=12353337
+  version: 12.1.2.Final
+  maven_latest: 12.1.2.Final
+  release_date: 2021/05/04
+  fixed_issues: https://issues.redhat.com/secure/ReleaseNote.jspa?projectId=12310799&version=12356461
   codename: Taedonggang
   whats_new: Implicit Authorization, Spring Protobuf caches, Automatic Cross Site Replication, YAML Configuration, Documentation improvement
   download:
-    server: https://downloads.jboss.org/infinispan/12.1.1.Final/infinispan-server-12.1.1.Final.zip
-    wfmodules: https://downloads.jboss.org/infinispan/12.1.1.Final/infinispan-wildfly-modules-12.1.1.Final.zip
-    docker: quay.io/infinispan/server:12.1.1.Final
-    docker_native: quay.io/infinispan/server-native:12.1.1.Final
-    docker_cli: quay.io/infinispan/cli:12.1.1.Final
+    server: https://downloads.jboss.org/infinispan/12.1.2.Final/infinispan-server-12.1.2.Final.zip
+    wfmodules: https://downloads.jboss.org/infinispan/12.1.2.Final/infinispan-wildfly-modules-12.1.2.Final.zip
+    docker: quay.io/infinispan/server:12.1.2.Final
```

Comment:
@ryanemerson can we use `quay.io/infinispan/server:12.1`? 
just to avoid updating everytime xD

Reply:
i prefer not because the image does not get updated locally if you don't ask for the minor

Reply:
Hmm it should if you call `docker pull` again.

Reply:
sometimes it caches images and it does not upgrade unless you force. at least i've experienced that myself

Reply:
`docker run` never updates the image, no one expects to pull the latest version.